packages/strapi-helper-plugin/lib/src/components/NavLink/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ *
+ * NavLink
+ *
+ */
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { isObject } from 'lodash';
+import cn from 'classnames';
+
+import styles from './styles.scss';
+
+function NavLink(props) {
+  let content = props.children;
+
+  if (typeof props.message === 'string') {
+    content = props.message;
+  }
+
+  if (isObject(props.message) && props.message.id) {
+    content = <FormattedMessage id={props.message.id} />;
+  }
+
+  let icon = <i className={`fa ${props.icon}`} />;
+
+  if (props.icon === 'layout') {
+    icon = <i className={cn(styles.fa, styles.layout)} />;
```

Comment:
I'm not sure the layout class exist in your `styles.scss` file therefore I don't think it's needed.